base/char.jl

```diff
@@ -45,8 +45,37 @@ isless(x::Integer, y::Char) = isless(x, UInt32(y))
 bswap(x::Char) = Char(bswap(UInt32(x)))
 
 print(io::IO, c::Char) = (write(io, c); nothing)
+
+const hex_chars = UInt8['0':'9';'a':'z']
```

Comment:
That's quite an extended HEX range. Any reason not to write `UInt8['0':'9';'a':'f']`?